### DISCUSSION TITLE
Brand new install, web login asking for username and password, I never set one up, nothing in logs

### DISCUSSION THREAD

**Discussion by bobloadmire** [nextcloud/all-in-one#2644] [Questions] [answered]
I'm at the login page (not the AIO page on port 8443, the regular login page) and its asking for username and password. I never set one up during the installation process, there doesn' seem to be a default one. I checked logs for any info on what it might be, nothing there on the NC container, or the master container. What do I do?

**Comment by saleh-mir** [comment]
I have the same issue!

**Reply by Zoey2936** [reply, marked as answer]
![grafik](https://github.com/nextcloud/all-in-one/assets/75573284/d77a5a36-f4dc-45c6-b0d1-4e4f8d7801d6)
did you seen that?
